### PR TITLE
(fix) O3-3398: Combine the translation for Create New Patient/ Edit Patient

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary
Create New Patient translation was broken into 2 words: Create New and Patient
The same was the case for Edit Patient: Edit and Patient

## Screenshots
None

## Related Issue
https://issues.openmrs.org/browse/O3-3398

### CODE REVIEW

File: packages/esm-patient-registration-app/translations/en.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "configure": "Configure",
   "configureIdentifiers": "Configure identifiers",
   "contactSection": "Contact Details",
-  "createNew": "Create New",
+  "createNewPatient": "Create New Patient",
```
Comment (denniskigen) [inline packages/esm-patient-registration-app/translations/en.json:18]:
```suggestion
  "createNewPatient": "Create new patient",
```

Prefer sentence-case. See https://zpl.io/VDNg9BO and https://carbondesignsystem.com/guidelines/content/writing-style#capitalization.

File: packages/esm-patient-registration-app/translations/en.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
   "discardModalBody": "The changes you made to this patient's details have not been saved. Discard changes?",
   "discardModalHeader": "Confirm Discard Changes",
   "dobToggleLabelText": "Date of Birth Known?",
-  "edit": "Edit",
   "editIdentifierTooltip": "Edit",
+  "editPatient": "Edit Patient",
```
Comment (denniskigen) [inline packages/esm-patient-registration-app/translations/en.json:31]:
```suggestion
  "editPatient": "Edit patient",
```

Comment (denniskigen) [reply packages/esm-patient-registration-app/translations/en.json:31]: Ditto

File: packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx [hunk 3]
```diff
@@ -158,9 +158,7 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
           <div className={styles.formContainer}>
             <div>
               <div className={styles.stickyColumn}>
-                <h4>
-                  {inEditMode ? t('edit', 'Edit') : t('createNew', 'Create New')} {t('patient', 'Patient')}
-                </h4>
+                <h4>{inEditMode ? t('editPatient', 'Edit Patient') : t('createNewPatient', 'Create New Patient')}</h4>
```
Comment (denniskigen) [inline packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx:161]:
```suggestion
                <h4>{inEditMode ? t('editPatientDetails', 'Edit patient details') : t('createNewPatient', 'Create new patient')}</h4>
```

https://zpl.io/agm3dOA

**Review by denniskigen — COMMENTED** [review]
Thanks for this, @vasharma05. I've left some suggestions about casing and a change to the Edit patient translation string.

**Comment by vasharma05** [comment]
Changes done @denniskigen!
Thanks for the catch!

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @vasharma05.

**Pull request merged**